services/admin/src/store.ts

```diff
@@ -105,6 +105,7 @@ export class Store {
    */
   deleteCvrs(): void {
```

Comment:
What do you think of renaming this `deleteCvrsAndCvrFiles` for clarity? Since other methods also make the distinction between CVRs and CVR files